spec/rubocop/cop/rspec/is_expected_specify_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::IsExpectedSpecify, :config do
+  it 'registers an offense when using `specify` and one-liner style' do
+    expect_offense(<<~RUBY)
+      specify { is_expected.to be_truthy }
+      ^^^^^^^ Use `it` instead of `specify`.
+      specify { are_expected.to be_falsy }
+      ^^^^^^^ Use `it` instead of `specify`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      it { is_expected.to be_truthy }
+      it { are_expected.to be_falsy }
+    RUBY
+  end
+
+  it 'registers an offense when using `specify` and one-liner with metadata' do
+    expect_offense(<<~RUBY)
+      specify "pending", :pending
+      ^^^^^^^ Use `it` instead of `specify`.
+    RUBY
+  end
+
+  it 'does not register an offense when using `specify` ' \
+     'and not one-liner style' do
+    expect_no_offenses(<<~RUBY)
+      specify { expect(sqrt(4)).to eq(2) }
+    RUBY
+  end
+
+  it 'does not register an offense when using `specify` and multi line' do
+    expect_no_offenses(<<~RUBY)
+      specify do
+        # this is a comment
```

Comment:
I think even without this comment it is not considered a one-liner, or is it?

Reply:
Indeed it is. I removed the comment.